frontend/src/components/chat-interface.tsx

```diff
@@ -125,6 +128,23 @@ export function ChatInterface() {
             </ChatMessage>
           ),
         )}
+
+        {messages.length > 2 &&
+          !isErrorMessage(lastMessage) &&
+          lastMessage.sender === "assistant" && (
+            <div className="mb-2">
+              <SuggestionItem
+                suggestion={{
+                  label: "Push to GitHub",
+                  value:
+                    "Please push the changes to GitHub and open a pull request.",
+                }}
+                onClick={(value) => {
+                  handleSendMessage(value, []);
+                }}
+              />
+            </div>
+          )}
```

Comment:
I think we need to make this "Download Zip" if you're not connected to GH

Reply:
Right